### PR TITLE
move features link to features teaser section

### PR DESCRIPTION
**Pull request by ronso0**
currently it's in the Source Code container :shrug: 

image

### CODE REVIEW

**Comment by Swiftb0y** [comment]
you'll probably also want to change the enclosing div container to be `thirds` (or something idk, not familiar with the css framework) instead of `quarters` so the buttons are still centered. 
image

**Comment by ronso0** [comment]
yes, I'm still tweaking it, have no clue about the css rules here either.

File: theme/templates/pages/index.html [hunk 1]
```diff
@@ -83,14 +83,20 @@ <h5 class="docs-header">{% trans %}Vinyl Record Control{% endtrans %}</h5>
         {% trans %}Use turntables with timecode vinyl records to control playback and scratch your digital music files as if they were pressed on vinyl. Mix music vinyls into your set by toggling vinyl passthrough mode.{% endtrans %}
       </p>
     </div>
+
+    <div>
+      <p>
+        {& trans %}See dozens more features or download Mixxx and start DJing now!{% endtrans %}
```
Comment (Swiftb0y) [inline theme/templates/pages/index.html]: typo ;)
```suggestion
        {% trans %}See dozens more features or download Mixxx and start DJing now!{% endtrans %}
```

**Comment by ronso0** [comment]
why is the Download button blue in your screen?
should be orange
image

**Comment by Swiftb0y** [comment]
> why is the Download button blue in your screen?

That has always been like that for me for deploy-previews. Isn't that a feature? It's fine when I build locally and fine on prod.

**Comment by ronso0** [comment]
> you'll probably also want to change the enclosing div container to be thirds

Hmm, will test. maybe the spcing is too much and the buttons look lost.

**Comment by Swiftb0y** [comment]
> That has always been like that for me for deploy-previews. Isn't that a feature? It's fine when I build locally and fine on prod.

Ah, no. I see. It's dark-reader which is disabled for localhost as well as plain mixxx.org, but it does interfere in the deploy previews.

**Comment by ronso0** [comment]
> > you'll probably also want to change the enclosing div container to be thirds
> 
> Hmm, will test. maybe the spcing is too much and the buttons look lost.

The buttons are great in the Source section but I think here an inline link suits better.

**Review by ywwg — APPROVED** [review]

**Pull request merged**